Cargo.toml

```diff
@@ -100,3 +104,69 @@ syslog      = "7.0"
 [features]
 default = []
 tls = ["tokio-rustls", "rustls-pemfile"]
+
+# TODO: Do we want this?
```

Comment:
So Routinator and Krill both have this set to `panic = "abort"`. The rationale given in the Krill `cargo.toml` is "# Make sure that Krill crashes on panics, rather than losing threads and limping on in a bad state". I.e. a panic in a particular thread could leave the application in a bad state so it is better to exit completely. That seems sensible, and clearly as we already do it for at least two projects we've discussed this before and concluded this was the best approach. Does anyone have a counter argument here? The only one I can think of is not based on fact, which is I'm wondering does panic without abort allow Drop handlers to run which if not run could cause even more mess, such as leaving persisted data in an inconsistent state?

Reply:
I think abort is best. And cascade should be able to handle crashes. The computer/VM may also crash. That should not affect the consistency of state. Mimir seems to be an exception where continuing works fine.